File: client/src/libs/auth/data-access/user-menu.contributor.ts
import { MenuContributor, MenuConfigurationContext, StandardMenus } from '@cheetah/shared/core';

export class UserMenuContributor implements MenuContributor {
  order = 1000;

  configureMenu(ctx: MenuConfigurationContext): void {
    ctx.getOrCreate(StandardMenus.USER)
       .addItem({
         id:       'logout',
         label:    'Выйти',
         icon:     'pi pi-sign-out',
         order:    100,
         url:      '/logout',
       });
  }
}

File: src/Cheetah.Frontend.Auth/Navigation/UserMenuContributor.cs
using Cheetah.Frontend.Navigation.Constants;
using Cheetah.Frontend.Navigation.Extensions;
using Cheetah.Frontend.Navigation.Services.Abstractions;

namespace Cheetah.Frontend.Auth.Navigation;

public class UserMenuContributor : IMenuContributor
{
    public Task ConfigureMenuAsync(MenuConfigurationContext context)
    {
        var userMenu = context.GetOrCreateMenu(StandardMenus.User);

        userMenu.AddItem(
            id: "Logout",
            name: "Logout",
            iconName: "box-arrow-right",
            url: "/logout",
            order: 100);

        return Task.CompletedTask;
    }
}
